ServerSide/yoty/WebApi/Controllers/YotyController.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) YOTY Corporation and contributors. All rights reserved.
+
+namespace YOTY.Service.WebApi.Controllers
+{
+    using System.Linq;
+    using System.Security.Claims;
+    using Microsoft.AspNetCore.Mvc;
+
+    public class YotyController: ControllerBase
+    {
+        protected string GetUserId()
+        {
+            return 
+                User
+                .Claims
+                .FirstOrDefault(i => i.Type == ClaimTypes.NameIdentifier)
+                ?.Value ;
+        }
+
+        protected string GetRequestUserId()
+        {
+            return User.FindFirst(ClaimTypes.NameIdentifier).Value;
+        }
+    }
+}
```

Comment:
Why are there 2 different methods to get the ID?

Reply:
סתם .. לא משתמשים בראשונה

Reply:
שיניתי